app/server/api/user_api.py

```diff
@@ -231,6 +231,27 @@ def post(self, user_id):
             }
             return make_response(jsonify(response_object)), 400
 
+class PasswordResetAPI(MethodView):
+    @requires_auth(allowed_roles={'ADMIN': 'superadmin'})
```

Comment:
It seems funny to superadmin-gate this, since technically anybody (even not logged in) can request a password reset email. Reckon that this endpoint nuke the TFA token as well?

Reply:
Hm - I think super admins shouldn't have the ability to reset TFA token - only sempo admins